src/app/purchase/AuctionRanking.tsx

```diff
@@ -0,0 +1,62 @@
+interface AuctionRankingProps {
+  maxPrice: number;
+}
+
+const AuctionRanking = (props: AuctionRankingProps) => {
+  const commaPrice = props.maxPrice.toLocaleString();
```

Comment:
```suggestion
const AuctionRanking = ({maxPrice}: AuctionRankingProps) = > {
const commaPrice = maxPrice.toLocaleString();
```
해당 부분을 구조분해할당을 사용하게 된다면 가독성이 더 올라갈거같습니다!